### ISSUE TITLE
aggregating calendars

### ISSUE THREAD

**Issue by KiaraGrouwstra** [nextcloud/calendar#4770]
### Is your feature request related to a problem? Please describe.

i would like to be able to have different groups collaborate on their own group calendars, then allow someone with access to the distinct calendars to aggregate these into a single entity to be published (read-only feed) to consumers.

as is, it is tough to decouple subscribers from collaborators to such an end.
either everyone would be collaborating on one and the same calendar, while they might not care about the other groups, or the calendars would have to be published separately, which means our organization cannot offer calendar feed consumers one over-arching feed.

### Describe the solution you'd like

allow composing calendars into a feed/calendar, such that collaborators and consumers can be decoupled

### Describe alternatives you've considered

having users subscribe to various calendars

### Additional context

cf. [similar question](https://help.nextcloud.com/t/aggregation-of-multiple-calendars-into-one/31938)

**Comment by KiaraGrouwstra** [comment]
sorry, this seems tackled as per the [docs](https://docs.nextcloud.com/server/latest/user_manual/en/groupware/calendar.html#publishing-a-calendar):
> Multiple calendars can be shared together by adding their unique tokens to the end of an embed link. Individual tokens can be found at the end of each calendar’s public link. The full address will look like `https://cloud.example.com/index.php/apps/calendar/embed/<token1>-<token2>-<token3>`

**Comment by KiaraGrouwstra** [comment]
on second look, this seems to work for links as the above, tho not for links like `https://cloud.example.com/remote.php/dav/public-calendars/<token1>-<token2>?export`.
i guess the request here might then be to extend this support to such links as well.

**Comment by cyfrax** [comment]
Yes please, combination doesn't work for ical export. We have 10 calendards managed by different individuals and need to show them aggregated as one. This would be so great. Thank you.

**Comment by tobiasBora** [comment]
I would also have a use case for a similar aggregation functionality that the above proposition does not solve: I have many different calendars coming from various sources (e.g. my university publishes multiple calendars, e.g. with important admin dates, with my courses in section XX, with my courses in section YY + I have calendars shared by other institutions with regular meetings etc) and I'd like to merge them all into a single read-only calendar to share it with my wife, and to sync it easily with my other devices (phone, thunderbird…). Note, of course, that the above solution involving `<token1>-<token2>…` does **not** work for this use case, as the **URL should be fixed**, otherwise I would need to send a new link everytime I send a new calendar. But I guess one could easily re-use this, for instance to create an alias so that a single `tokenN` redirects to a (customizable) url `<token1>-<token2>`…

**Comment by SebastianKrupinski** [comment]
Related: https://github.com/nextcloud/calendar/issues/7412

**Comment by SebastianKrupinski** [comment]
Related: https://github.com/nextcloud/calendar/issues/7641
